HJM.Chip8/HJM.Chip8.CPU/Instructions/Instruction.cs

```diff
@@ -0,0 +1,25 @@
+﻿using HJM.Chip8.CPU.Changes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HJM.Chip8.CPU.Instructions
+{
+    /// <summary>
+    /// Implement one instruction from the instruction set
+    /// </summary>
+    public abstract class Instruction
+    {
+        /// <summary>
+        /// Description of what the instruction does
+        /// </summary>
+        string Description { get; set; } = string.Empty;
```

Comment:
This could also be abstract property so every instruction would have to implement it and provde their own description.

Reply:
Yeah that definitely makes more sense now that I think about it.